docs/getting_started.md

```diff
@@ -266,7 +266,7 @@ documentation](https://cmake.org/documentation/).
 
 | Option               | Description |
 |----------------------|-------------|
-| CMAKE_BUILD_TYPE     | Build type, e.g. Debug or Release; affects compiler flags |
+| CMAKE_BUILD_TYPE     | Build type, e.g. Debug or Release; affects compiler flags <br/> (if not specified **`RelWithDebInfo`** will be used as a default) |
```

Comment:
Nitpicking: is the manual `<br/>` really needed?

Reply:
It is, yes. Otherwise it scrolls horizontally.